tests/spec/table_from_file/test_table_from_file.py

```diff
@@ -18,10 +16,18 @@
         """,
 }
 
+csv_data = [
+    (1, 100),
+    (3, 300),
+]
+
 
-@pytest.mark.xfail
-def test_table_from_file(spec_test):
-    path = Path("")
+def test_table_from_file_csv(spec_test, tmp_path):
+    path = tmp_path / "test_table_from_file.csv"
+    with path.open("w") as f:
+        writer = csv.writer(f)
+        writer.writerow(["patient_id", "n"])
+        writer.writerows(csv_data)
```

Comment:
Alternatively, pass a string to `path.write_text`. For example, see:

https://github.com/opensafely-core/databuilder/blob/6bb616acb9187ff8f85e8c6fcdbe29420f27ef15/tests/unit/test_codes.py#L18-L27